docs/ZFS-RAID-CONFIGURATION.md

```diff
@@ -0,0 +1,30 @@
+# EVE-OS supported RAID configuration(s)
+
+| RAID Level | Separate EVE-OS disk | Minimum number of disks for RAID | Example Grub Configuration              |
```

Comment:
"Minimum number of disks for RAID" doesn't make sense in the first case. "Minimum number of disks" would make more sense.

Reply:
But isn't that minimum number of disks to get redundancy ? That's why I used RAID there.

Reply:
But in the first row there is no redundancy - just a single disk. And RAID0 aka striping doesn't provide redundancy.

Reply:
I removed  RAID in the latest commit. It will be just "Minimum number of disks required"